burst/README.md

```diff
@@ -0,0 +1,87 @@
+# Burst Metrics Service
+
+Burst metrics service is responsible for reading HPA metrics from a Kubernetes(k8s) API and return a formatted burst metrics header. It is implemented as a REST service.
+
+Currently it will capture current pod count and target pod count from an HPA.
+
+## Deployment/usage
+
+TODO: add recently pushed `make` usage
+
+## API Endpoints
+
+### `/burstmetrics/{namespace}/{HPA}`
+
+This is the only endpoint exposed by this service.
+This endpoint will check the k8s `{namespace}` for the specified `{HPA}`.
+
+If the `{HPA}` is found and is accessible, it will return the metrics below in a formatted fashion:
+
+- service=namespace/HPA-name
+- current-load=current pod count
+- max-load=target hpa pod count
+- target-load=80% of max-load <!-- WIP:Configure the percentage -->
+
+If it can't find the `{HPA}` or is inaccessible, or any exception occurs during the API call, it will simply return `HTTP 204 No Content`.
+
+#### Example 1: 200 OK
+
+```http
+HTTP/1.1 200 OK
+content-type: text/plain; charset=utf-8
+date: Mon, 23 Aug 2021 16:29:51 GMT
+server: istio-envoy
+transfer-encoding: chunked
+x-envoy-upstream-service-time: 1
+
+service=default/ngsa, current-load=2, target-load=5, max-load=6
+```
+
+#### Example 2: 204 No Content
+
+```http
+HTTP/1.1 204 No Content
+date: Mon, 23 Aug 2021 16:30:26 GMT
+server: istio-envoy
+x-envoy-upstream-service-time: 16
+```
+
+### RBAC requirements
+
+In-cluster configuration (as well as default kube-config) for this service requires permissions to call the `autoscaling` API. Shown below is an example of a `ClusterRole` with `autoscaling` API groups added.
+
```

Comment:
I would put a link to the file rather than the yaml so the docs don't get out of sync